cli/Cargo.toml

```diff
@@ -23,8 +23,7 @@ sp-runtime = { git = "https://github.com/paritytech/substrate", branch = "master
 sc-client-api = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sc-client-db = { git = "https://github.com/paritytech/substrate", branch = "master" }
 sc-executor = { git = "https://github.com/paritytech/substrate", branch = "master" }
-service = { package = "polkadot-service", path = "../service", default-features = false, optional = true }
-service-new = { package = "polkadot-service-new", path = "../node/service", default-features = false, optional = true }
+service = { package = "polkadot-service-new", path = "../node/service", default-features = false, features = [ "full-node" ] }
```

Comment:
dumb question: with this, it's not possible to opt-out of the `full-node` feature, how would it work with the light client?

Reply:
Not dumb. Maybe I need to introduce a `full-node` feature in the CLI.

Reply:
The feature name is misleading. This feature is used to compile the node for the browser, where we only run the light client. But the "native" light client can be run with this feature enabled.

Reply:
Maybe it's worth renaming it to `native-client` to differentiate between `native` and `browser` environments.

Reply:
Updated in c0e9e437.

Reply:
@coriolinus did you read my comment above? Propagating the feature makes no sense. Cli will not be used by the browser.

Reply:
@bkchr what is the `browser` feature used for then?

Reply:
I see, I assumed some different structure.

Reply:
So, it was before already propagated, but using the `service` feature.

Reply:
TBH, I don't know if the `browser` feature of polkadot's cli was ever tested (cc @expenses), AFAIK people experimented with the browser demo directly in substrate.

So it may be out of scope of this PR, we can fix it later.

Reply:
At this point I am a bit confused. @bkchr No, I didn't see your comment until after I'd added the patch propagating the feature. What's your recommended course of action at this point?

Reply:
TLDR your commit is fine